doc/guides/contributing/code-of-conduct.md

```diff
@@ -27,7 +27,7 @@ spirit of the [Code of Conduct][]. Failure, or unwillingness, to do so will
 result in contributions being respectfully declined.
 
 A *bad actor* is someone who repeatedly violates the *spirit* of the Code of
```

Comment:
Optional suggestion:

```suggestion
A *bad actor* is someone who repeatedly violates the spirit of the Code of
```

(Using italics once to highlight the term being defined, and then using it again in the same sentence for emphasis seems suboptimal. Remove the one for emphasis as its unnecessary.)